internal/mode/static/nginx/config/servers_test.go

```diff
@@ -1221,17 +1222,17 @@ func TestCreateLocationsRootPath(t *testing.T) {
 				{
 					Path:            "/path-1",
 					ProxyPass:       "http://test_foo_80$request_uri",
-					ProxySetHeaders: baseHeaders,
+					ProxySetHeaders: httpBaseHeaders,
 				},
 				{
 					Path:            "/path-2",
 					ProxyPass:       "http://test_foo_80$request_uri",
-					ProxySetHeaders: baseHeaders,
+					ProxySetHeaders: httpBaseHeaders,
 				},
 				{
 					Path:            "/",
 					ProxyPass:       "http://test_foo_80$request_uri",
-					ProxySetHeaders: baseHeaders,
+					ProxySetHeaders: httpBaseHeaders,
 				},
 			},
```

Comment:
i think we should add a test case where both grpc ad http headers are set

Reply:
That can't happen in this case, a path is either gRPC _or_ HTTP.